packages/server/tests/acceptance/rpc_batch2.spec.ts

```diff
@@ -142,7 +142,8 @@ describe('@api-batch-2 RPC Server Acceptance Tests', function () {
             expect(estimatedGas).to.equal(expectedRes);
         });
 
-        it('@release should execute "eth_estimateGas" for existing account', async function() {
+        // Skip this test for now because of bug in mirror-node https://github.com/hashgraph/hedera-mirror-node/issues/6612 in Additional bug fixes
```

Comment:
Please open a ticket to fix this so we don't miss it

Reply:
Yes, opened here https://github.com/hashgraph/hedera-json-rpc-relay/issues/1623